### PR TITLE
Wubzy's themes and README updates

### PR DESCRIPTION
**Pull request by WubzyGD**
This PR adds a large handful of my own themes, as well as updates some README links and includes more info in the main README and then Community folder README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,7 +60,7 @@ Any issues can be expressed by:
 
 * [Joining the Discord server](https://discord.gg/FdZhbjY)
 * [Opening an issue on GitHub](https://github.com/puckzxz/NotAnotherAnimeTheme/issues)
-* Messaging me on Discord at puckzxz#2080
+* Messaging me on Discord at [puckzxz#2080](https://discord.com/users/135554522616561664)
```
Comment (puckzxz) [inline README.md:63]: I think this should just be removed now since we have a popular support server

Comment (WubzyGD) [reply README.md]: Fair enough, I can agree with that. Any way to reject that specific diff?

Comment (puckzxz) [reply README.md]: Already removed 👍

Comment (WubzyGD) [reply README.md]: Ah nvm lol

**Review by puckzxz — APPROVED** [review]
LGTM 👍

**Pull request merged**